extensions/amp-position-observer/validator-amp-position-observer.protoascii

```diff
@@ -39,6 +39,10 @@ tags: {  # <amp-position-observer>
     value_regex: "^([0]*?\\.\\d*$|1$|0$)|([0]*?\\.\\d*|1|0)\\s{1}([0]*?\\.\\d*$|1$|0$)"
   }
   attrs: { name: "target" }
+  attrs: {
+    name: "once"
```

Comment:
could you update the validation test files (https://github.com/ampproject/amphtml/blob/master/extensions/amp-position-observer/0.1/test/validator-amp-position-observer.html) ( running `gulp validator --update_tests will auto update the `.out` file)

Reply:
Done.